evaluation/utils/version_control.sh

```diff
@@ -39,8 +39,8 @@ checkout_original_branch() {
     git checkout $current_branch
 }
 
-get_agent_version() {
+get_openhands_version() {
     # IMPORTANT: Because Agent's prompt changes fairly often in the rapidly evolving codebase of OpenHands
     # We need to track the version of Agent in the evaluation to make sure results are comparable
-    AGENT_VERSION=v$(poetry run python -c "import openhands.agenthub; from openhands.controller.agent import Agent; print(Agent.get_cls('$AGENT').VERSION)")
+    OPENHANDS_VERSION=v$(poetry run python -c "import openhands.agenthub; from openhands.controller.agent import Agent; print(Agent.get_cls('$AGENT').VERSION)")
```

Comment:
This still gets agent version though 😄

Reply:
I think we want this https://github.com/All-Hands-AI/OpenHands/blob/793e142c4a4656543497a8c98935a1558a090a5b/openhands/__init__.py#L6

Reply:
ahhh sorry forgot to push second commit!

Reply:
Switched to `get_version` - seems more robust that way